Command/LoadDataFixturesDoctrineODMCommand.php

```diff
@@ -82,37 +111,51 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        if ($dirOrFile) {
-            $paths = is_array($dirOrFile) ? $dirOrFile : [$dirOrFile];
-        } elseif ($bundles) {
-            $paths = [$this->getKernel()->getRootDir().'/DataFixtures/MongoDB'];
-            foreach ($bundles as $bundle) {
-                $paths[] = $this->getKernel()->getBundle($bundle)->getPath();
+        if ($services === false) {
```

Comment:
This code path also needs to trigger a deprecation notice explaining that using `--bundles` and `--fixtures` is deprecated and will be removed in 4.0.